### PR TITLE
Code for Shell Sort in Python

### PR DESCRIPTION
**Pull request by rahulkumaran**
This is a code for Shell Sort written in Python.
For testing once can give any list of numbers arranged in any order.
Also, I assigned the task of writing the code to myself as there was no code already for the implementation of Shell Sort!

### CODE REVIEW

**Comment by rahulkumaran** [comment]
@prateekiiest rectified the mistake !

File: Competitive Coding/Sorting/Shell_Sort/shellsort.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		gapInsertionSort(test,startposition,sublistcount)
+	print("After increments of size",sublistcount,"The list is",test)
+	sublistcount = sublistcount // 2
+
+def gapInsertionSort(test,start,gap):
+	for i in range(start+gap,len(test),gap):
+		currentvalue = test[i]
+		position = i
+		while(position>=gap and test[position-gap]>currentvalue):
+			test[position]=test[position-gap]
```
Comment (prateekiiest) [inline Competitive Coding/Sorting/Shell_Sort/shellsort.py:22]: document the code

**Review by prateekiiest — CHANGES_REQUESTED** [review]

**Comment by prateekiiest** [comment]
Why is Ceasar Cipher showing up here ?

**Comment by prateekiiest** [comment]
Your PR is not working as expected.
Please open a new branch for each PR.

That way it will get solved. If not try to edit the master repo. That will be better

File: Competitive Coding/Sorting/Shell_Sort/shellsort.py (hunk 1 above)
Comment (rahulkumaran) [reply Competitive Coding/Sorting/Shell_Sort/shellsort.py:22]: Okay I'll do that!

**Comment by rahulkumaran** [comment]
And i created a new branch, not able to figure out why the Caesarcipher code came in this branch ! Anyway I'll try rectifying

**Pull request closed without merging**